community-operators/bundles/camel-k/bundle.0.0.1.yaml

```diff
@@ -0,0 +1,322 @@
+      kind: CustomResourceDefinition
+      metadata:
+        name: integrationcontexts.camel.apache.org
+        labels:
+          app: "camel-k"
+      spec:
+        group: camel.apache.org
+        scope: Namespaced
+        version: v1alpha1
+        names:
+          kind: IntegrationContext
+          listKind: IntegrationContextList
+          plural: integrationcontexts
+          singular: integrationcontext
+          shortNames:
+          - ictx
+        additionalPrinterColumns:
+          - name: Phase
+            type: string
+            description: The IntegrationContext phase
+            JSONPath: .status.phase
+          - name: Type
+            type: string
+            description: The IntegrationContext type
+            JSONPath: .metadata.labels.camel\.apache\.org\/context\.type
+          - name: Image
+            type: string
+            description: The IntegrationContext image
+            JSONPath: .status.image
+    - apiVersion: apiextensions.k8s.io/v1beta1
+      kind: CustomResourceDefinition
+      metadata:
+        name: integrationplatforms.camel.apache.org
+        labels:
+          app: "camel-k"
+      spec:
+        group: camel.apache.org
+        scope: Namespaced
+        version: v1alpha1
+        names:
+          kind: IntegrationPlatform
+          listKind: IntegrationPlatformList
+          plural: integrationplatforms
+          singular: integrationplatform
+          shortNames:
+          - ip
+        additionalPrinterColumns:
+          - name: Phase
+            type: string
+            description: The IntegrationPlatform phase
+            JSONPath: .status.phase
+    - apiVersion: apiextensions.k8s.io/v1beta1
+      kind: CustomResourceDefinition
+      metadata:
+        name: integrations.camel.apache.org
+        labels:
+          app: "camel-k"
+      spec:
+        group: camel.apache.org
+        scope: Namespaced
+        version: v1alpha1
+        names:
+          kind: Integration
+          listKind: IntegrationList
+          plural: integrations
+          singular: integration
+          shortNames:
+          - it
+        additionalPrinterColumns:
+          - name: Phase
+            type: string
+            description: The Integration phase
+            JSONPath: .status.phase
+          - name: Context
+            type: string
+            description: The IntegrationContext to use
+            JSONPath: .status.context
+  clusterServiceVersions: |-
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: ClusterServiceVersion
+      metadata:
+        name: camel-k-operator.v0.1.0
+        namespace: palceholder
+        annotations:
+          categories: "apache, camel-k, kubernetes, serverless, microservices, integration, code"
+          certified: "false" 
+          containerImage: docker.io/apache/camel-k
+          createdAt: 2019-01-20T12:00:00Z
+          support: Apache Camel Community
+          alm-examples: |-
```

Comment:
Please update this block to treat it as a string.

See https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines